haystack/components/preprocessors/document_cleaner.py

```diff
@@ -18,10 +18,11 @@ class DocumentCleaner:
     """
     Cleans the text in the documents.
 
-    Cleans up text documents by removing extra whitespaces, empty lines, specified substrings, regexes,
+    It removes extra whitespaces,
+    empty lines, specified substrings, regexes,
     page headers and footers (in this order).
 
-    Usage example:
+    ### Usage example:
```

Comment:
```suggestion
    ### Usage example:
    
```

I believe there needs to be an extra line after a header for it to be rendered correctly